Lib/glyphsets/encodings/canadian-aboriginal_unique-glyphs.nam

```diff
@@ -719,4 +719,19 @@
 0x18F3 CANADIAN SYLLABICS BEAVER DENE L
 0x18F4 CANADIAN SYLLABICS BEAVER DENE R
 0x18F5 CANADIAN SYLLABICS CARRIER DENTAL S
-0x25CC DOTTED CIRCLE
```

Comment:
Is removal of DOTTED CIRCLE intentional?

Reply:
It wasn't, but on the other hand I don't believe that Canadian Syllabics uses any nonspacing marks.